joanne.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Joanne's Page</title>
+</head>
+<body>
+  <img src="https://media.giphy.com/media/l41lN5w0TZIN0YwNy/giphy.gif"</img>
```

Comment:
Please refer to this documentation for implementation of the `<img>` tag.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img

Change to the following:  
```html
  <img src="https://media.giphy.com/media/l41lN5w0TZIN0YwNy/giphy.gif">
```